cli/delete.go

```diff
@@ -32,7 +32,7 @@ var (
 		},
 		cli.StringFlag{
 			Name:  "obj.size",
-			Value: "1KB",
+			Value: "1MiB",
```

Comment:
There shouldn't be a need to upload this big objects since it is only for delete tests.

Reply:
The reason I did that was for faster uploads @klauspost it's too slow.

Reply:
@harshavardhana How is increasing the size helping that?